third_party/npm_@ledgerhq_hw-app-eth/LICENSE

```diff
@@ -0,0 +1,245 @@
+                                 Apache License
```

Comment:
cc @fmarier on if this is the right way to add these licenses

Reply:
That's fine because it is actually not just the Apache-2.0 license. It's got an extra bit added at the end.

If it were the normal Apache-2.0 license, then you could re-use the file in `common/licenses/Apache-2.0`.

Reply:
done